src/app/components/PageLayout/NetworkSelector.tsx

```diff
@@ -84,7 +84,7 @@ const NetworkSelectorView: FC<NetworkSelectorViewProps> = ({ isOutOfDate, layer,
       {!isMobile && (
         <NetworkButton isOutOfDate={isOutOfDate} layer={layer} network={network} onClick={handleDrawerOpen} />
       )}
-      {!isTablet && network !== Network.mainnet && (
+      {!hasFixedLayer() && !isTablet && network !== Network.mainnet && (
```

Comment:
Avoid function calls, this probably stays the same for the whole session. So the returned value, can be static.

Reply:
Fixed